src/scheduler/peer_organizer.rs

```diff
@@ -34,10 +34,12 @@ pub enum Task {
     InsertPeer(HandshakeInfo),
     PenalPeer(PeerId, PeerPenal, String), //last is reason
     WaitForStatus(PeerId, MessageData),
+    GetBlocks(PeerId, MessageData),
```

Comment:
GetBlockHeaders?

Reply:
I started implementing the `(Get)BlockBodies` requests yesterday, and already changed the name of the enum variant to `InitialRequest` (i. e. not a response to a peer request) with a  message id parameter.